.github/workflows/cross-version-tests.yml

```diff
@@ -193,6 +201,14 @@ jobs:
       - uses: ./.github/actions/setup-java
         with:
           java-version: ${{ matrix.java }}
+      # TODO: Remove this step once we remove the pyspark constraint in requirements/constraints.txt
+      - name: Remove pyspark constraint
+        run: |
+          if [ "${{ matrix.package }}" = "pyspark" ]; then
+            sed -i '/^pyspark/d' requirements/constraints.txt
+            git add requirements/constraints.txt
+            git commit -m "Remove pyspark constraints for testing"
+          fi
```

Comment:
Features/tests relying on delta-spark still need the constraint, but pyspark cross-version tests don't.